src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -175,7 +178,7 @@ export default class ClientMetricReport {
     jitter: { transform: this.secondsToMilliseconds, type: SdkMetric.Type.RTC_MIC_JITTER_MS },
     packetsSent: { transform: this.countPerSecond, type: SdkMetric.Type.RTC_MIC_PPS },
     bytesSent: { transform: this.bitsPerSecond, type: SdkMetric.Type.RTC_MIC_BITRATE },
-    roundTripTime: { transform: this.identityValue, type: SdkMetric.Type.RTC_MIC_RTT_MS },
+    roundTripTime: { transform: this.secondsToMilliseconds, type: SdkMetric.Type.RTC_MIC_RTT_MS },
```

Comment:
The `type` is named as `RTC_MIC_RTT_MS`, which means the metrics should be converted to millisecond based.